code/src/Album.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { Artist } from './Artist'
+
+
+export const Album = (props) => {
+    if (props.albumType === "album") {
+        console.log(props)
```

Comment:
remember to remove console.logs  from your code after you finish debugging your code.